snapshot/lib/snapshot/options.rb

```diff
@@ -232,7 +237,23 @@ def self.available_options
                                      env_name: "SNAPSHOT_CLONED_SOURCE_PACKAGES_PATH",
                                      description: "Sets a custom path for Swift Package Manager dependencies",
                                      type: String,
-                                     optional: true)
+                                     optional: true),
+        FastlaneCore::ConfigItem.new(key: :only_testing,
```

Comment:
We may want to think of a different key besides `only_testing` 🤔 A name like this kind of implies a boolean so we might want to have the key name be a little more explicit for what it actually wants.

Reply:
Good point - I didn't think of that as original naming comes from Apple directly and `xcodebuild` command:
https://developer.apple.com/library/archive/technotes/tn2339/_index.html

So just to be consistent with naming used by Apple - I used exactly same parameter names.
But I agree that something like `tests_to_run` `tests_to_skip` would be much more self-explanatory compared to their naming. But question is what's better - consistency with parent commands or self-explanatory parameter names. So far I saw fastlane is using 1:1 naming with `xcodebuild` so it's easier to identify them.

Reply:
I would probably say consistency with _fastlane_ naming in this case. IMO, most users aren't familiar with the options in the commands that _fastlane_ actions get piped too (especially with xcodebuild since there are a lot of them there 😊 )

Reply:
Much appreciated ❤️